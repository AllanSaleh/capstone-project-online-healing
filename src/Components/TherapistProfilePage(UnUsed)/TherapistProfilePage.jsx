import React from 'react';

import ProfilePic from './Images/ProfilePic.svg';
import ProfileInputField from '../EditProfilePage/ProfileInputField';
import BirthDateIcon from './Images/BirthDateIcon.svg';

export default function TherapistProfilePage({ therapistID }) {
  // Grab the therapist data from firebase
  alert(`Chosen therapist is: ${therapistID}`);

  return (
    <div className="flex flex-col lg:flex-row px-sides pt-navbar lg:h-firstsection">
      <div className="w-full lg:w-2/5 p-4 mx-2 text-center">
        <button type="button" className="hover:cursor-pointer px-20 py-20 lg:py-0 lg:px-10">
          <img src={ProfilePic} alt="profile-pic" />
        </button>
      </div>
      <div className="w-full lg:w-3/5 p-4 mx-2">
        <h1 className="text-4xl lg:text-title uppercase">therapist profile</h1>
        <ProfileInputField title="full name" type="text" id="full-name" />
        <div className="flex my-6">
          <div htmlFor="bio" className="text-subtitle capitalize inline-block w-1/3">
            bio
          </div>
          <div className="w-2/3 h-64 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4">
            <textarea
              id="bio"
              name="bio"
              className="w-11/12 h-full outline-none py-3 resize-none"
            />
          </div>
        </div>
        <ProfileInputField title="birth date" type="text" id="birth-date" icon={BirthDateIcon} />
        <ProfileInputField title="phone number" type="text" id="phone-number" />
      </div>
    </div>
  );
}
