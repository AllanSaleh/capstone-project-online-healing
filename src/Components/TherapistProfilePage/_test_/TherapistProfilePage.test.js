import * as React from 'react';
import TherapistProfilePage from "../ThrapistProfilePage";
import * as renderer from "react-test-renderer";

it('Therapist Profile Page renders correctly', () => {
  const tree = renderer
    .create(<TherapistProfilePage/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});