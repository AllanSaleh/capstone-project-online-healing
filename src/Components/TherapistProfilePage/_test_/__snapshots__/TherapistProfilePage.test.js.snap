// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Therapist Profile Page renders correctly 1`] = `
<div
  className="flex flex-col lg:flex-row px-sides pt-navbar lg:h-firstsection"
>
  <div
    className="w-full lg:w-2/5 p-4 mx-2 text-center"
  >
    <button
      className="hover:cursor-pointer px-20 py-20 lg:py-0 lg:px-10"
      type="button"
    >
      <img
        alt="profile-pic"
        src="ProfilePic.svg"
      />
    </button>
  </div>
  <div
    className="w-full lg:w-3/5 p-4 mx-2"
  >
    <h1
      className="text-4xl lg:text-title uppercase"
    >
      therapist profile
    </h1>
    <div
      className="flex items-center my-6"
    >
      <label
        className="text-subtitle capitalize inline-block w-1/3"
        htmlFor="full-name"
      >
        full name
      </label>
      <div
        className="w-2/3 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
      >
        <input
          className="w-11/12 h-full outline-none py-3"
          id="full-name"
          name="full-name"
          type="text"
        />
      </div>
    </div>
    <div
      className="flex my-6"
    >
      <div
        className="text-subtitle capitalize inline-block w-1/3"
        htmlFor="bio"
      >
        bio
      </div>
      <div
        className="w-2/3 h-64 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
      >
        <textarea
          className="w-11/12 h-full outline-none py-3 resize-none"
          id="bio"
          name="bio"
        />
      </div>
    </div>
    <div
      className="flex items-center my-6"
    >
      <label
        className="text-subtitle capitalize inline-block w-1/3"
        htmlFor="birth-date"
      >
        birth date
      </label>
      <div
        className="w-2/3 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
      >
        <input
          className="w-11/12 h-full outline-none py-3"
          id="birth-date"
          name="birth-date"
          type="text"
        />
        <button
          type="button"
        >
          <img
            alt="profileIcon"
            src="BirthDateIcon.svg"
          />
        </button>
      </div>
    </div>
    <div
      className="flex items-center my-6"
    >
      <label
        className="text-subtitle capitalize inline-block w-1/3"
        htmlFor="phone-number"
      >
        phone number
      </label>
      <div
        className="w-2/3 inline-flex items-center border border-black border-opacity-20 rounded-lg pl-4"
      >
        <input
          className="w-11/12 h-full outline-none py-3"
          id="phone-number"
          name="phone-number"
          type="text"
        />
      </div>
    </div>
  </div>
</div>
`;
